sa/model.go

```diff
@@ -1426,3 +1427,38 @@ type pausedModel struct {
 	PausedAt       time.Time  `db:"pausedAt"`
 	UnpausedAt     *time.Time `db:"unpausedAt"`
 }
+
+type overrideModel struct {
+	LimitEnum int64     `db:"limitEnum"`
+	BucketKey string    `db:"bucketKey"`
+	Comment   string    `db:"comment"`
```

Comment:
See my comment above: since the comment column is nullable, this should technically be a `*string`. But I think it's better to make the column non-NULLable than to change this.